plugins/chat/spec/system/message_notifications_with_sidebar_spec.rb

```diff
@@ -238,45 +238,6 @@ def create_message(text: nil, channel:, creator: Fabricate(:user))
           end
         end
       end
-
```

Comment:
this test has been xited for a long time and is kinda irrelevant with my new spec